packages/core/src/components/group-item/group-item.scss

```diff
@@ -20,6 +20,7 @@ $focusBorderWidth: 1px;
   display: flex;
   min-height: 2.25rem;
   height: 2.25rem;
+  cursor: pointer;
```

Comment:
Setting the cursor on the button should suffice

Reply:
Yes, but in this way , we give an option to manipulate to change cursor.
and also ionic is following same approach:
https://github.com/ionic-team/ionic-framework/tree/main/core/src/components

Reply:
I get your argument, but we will develop a guide line how to handle cursors in the whole design system. Until that will don't want to change the this.